stock_request_submit/tests/test_stock_request_submit.py

```diff
@@ -2,52 +2,121 @@
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0).
 
 from odoo import fields
-
-from odoo.addons.stock_request.tests import test_stock_request
+from odoo.tests.common import TransactionCase
 
 from ..hooks import uninstall_hook
 
 
-class TestStockRequestSubmit(test_stock_request.TestStockRequest):
-    def setUp(self):
-        super().setUp()
-        expected_date = fields.Datetime.now()
```

Comment:
What is the reason for removing this?

Reply:
@victoralmau Because I change this to classmethod and I move `expected_date` to `# Data` section (https://github.com/OCA/stock-logistics-warehouse/pull/1594/commits/f49a6e1ebc9d2e78a321676d3ebbec7fed3a0c1f#diff-ddfb508ae723cf68a5b2e4a3be83aa2ff8c5f7948855e952e7c692a283dead06R25).

Reply:
Sorry, my comment is about extending `test_stock_request.TestStockRequest`